frontend/src/services/actual_profile_backend/actual_profile_backend.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Communicates with update 
+ */
+export class ProfileBackendService {
+  
+  loadProfile(id: string): Promise<string> {
+    /** TODO: Upon OAuth and GetProfile being implemented, fetch /get-profile. */
+    const returnMessage = "TODO: Implement loadProfile upon OAuth being implemented.";
+    return Promise.resolve(returnMessage);
+  }
+
+  /** TODO: Make sure you're updating person, not just adding them  */
```

Comment:
```suggestion
  /** TODO: Make sure you're updating person, not just adding them. */
```
Period at end of comment.